moped-editor/src/components/GridTable/helpers.js

```diff
@@ -31,7 +31,8 @@ export const generateEmptyFilter = () => {
  */
 export const checkIsValidInput = (filterParameter, type) => {
   // If we are testing a number type field with a non null value
-  if (type === "number" && !!filterParameter.value) {
+  // if (type === "number" && !!filterParameter.value) {
```

Comment:
Could you remove the commented code here? This change looks good to me! Just to confirm - we are validating `array` type inputs for number characters only because the search only supports single districts for now, right? A comment somewhere in here that documents how this works could be helpful to understand how this works in the future.

Edit: Just saw the later discussion about this. A spin off issue is great! Still think a comment would hold us over until that issue is prioritized.

Reply:
great call, thank you Mike! 

https://github.com/cityofaustin/atd-moped/pull/1277/commits/edc111ff7dc880f71648c3cfe644dff8857f5011